addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -606,6 +606,7 @@ var SnippetEditor = Widget.extend({
         const proms = sortBy(styles, "__order").map((style) => {
             return show ? style.onTargetShow() : style.onTargetHide();
         });
+        this.options.wysiwyg.snippetsMenu.reload_snippet_dropzones();
```

Comment:
Note: the custom event was removed with the [wysiwyg refactoring](https://github.com/odoo/odoo/commit/d7245d2abf528d093226c80e40975e63d61e8997) and that syntax used in replacement in `mass_mailing_html_field.js`.
In master, with the refactoring of SnippetsMenu to OWL, this is not possible anymore, so I switched to registering the function in `this.options.reloadSnippetDropzones` directly. So I could use that here too. Or 3rd option: re-add a custom event instead.
Let me know what you prefer @qsm-odoo